tsconfig.json

```diff
@@ -23,7 +23,7 @@
     "./gatsby-node.ts",
     "./plugins/**/*",
     "./src/**/*",
-    "./utils/typings/svg.d.ts"
+    "./utils/typings/*.d.ts"
```

Comment:
Was there a reason to not include all `*.dt.s` files?

Reply:
Probably not :wink: